drivers/CoreLED/tests/CoreLED_test.cpp

```diff
@@ -68,3 +68,49 @@ TEST_F(CoreLEDTests, setColorAndShowRandomColor)
 	coreled.setColor(color);
 	coreled.showColor();
 }
+
+TEST_F(CoreLEDTests, setAndShowColorThenHideColor)
+{
+	coreled.setColor(RGB::pure_green);
+	coreled.showColor();
+
+	RGB color = RGB::black;
+
+	{
+		InSequence seq;
+
+		EXPECT_CALL(spimock, write(frame::start)).Times(1);
+		EXPECT_CALL(spimock, write(testing::ElementsAre(_, color.red, color.green, color.blue))).Times(20);
+		EXPECT_CALL(spimock, write(frame::reset)).Times(1);
+		EXPECT_CALL(spimock, write(frame::end)).Times(1);
+	}
+
+	coreled.hideColor();
+}
+
+TEST_F(CoreLEDTests, hideColorWhenAlreadyHidden)
+{
+	coreled.hideColor();
+
+	EXPECT_CALL(spimock, write).Times(0);
+
+	coreled.hideColor();
+}
+
+TEST_F(CoreLEDTests, setAndShowColorWhenColorHidden)
+{
+	coreled.hideColor();
+
+	RGB color = RGB::pure_blue;
+	{
+		InSequence seq;
+
+		EXPECT_CALL(spimock, write(frame::start)).Times(1);
+		EXPECT_CALL(spimock, write(testing::ElementsAre(_, color.red, color.green, color.blue))).Times(20);
+		EXPECT_CALL(spimock, write(frame::reset)).Times(1);
+		EXPECT_CALL(spimock, write(frame::end)).Times(1);
+	}
+
+	coreled.setColor(color);
+	coreled.showColor();
```

Comment:
Ca marche aussi avec la suggestion?

```suggestion
	coreled.showColor(color);
```